src/Projection/PdoEventStoreProjector.php

```diff
@@ -501,7 +502,11 @@ private function fetchRemoteStatus(): ProjectionStatus
 EOT;
 
         $statement = $this->connection->prepare($sql);
-        $statement->execute([$this->name]);
+        try {
+            $statement->execute([$this->name]);
+        } catch (PDOException $exception) {
+            // ignore
```

Comment:
There are no error codes checked there. Shall I check for errors and throw custom exception ?